.github/workflows/maven.yml

```diff
@@ -32,5 +32,9 @@ jobs:
         java-version: 11
     - name: Style check
       run: 'mvn spotless:check'
-    - name: Run test suite
+    - name: Run test phase
       run: mvn test
+    - name: Upload coverage report to Codecov
+      uses: codecov/codecov-action@v1
+      with:
+        files: ./target/site/jacoco/jacoco.xml
```

Comment:
You can omit this. Codecov is clever enough to find the report on its own for Maven/Jacoco projects.

For most "standard" tools, Codecov is pretty good at just finding the report.

Reply:
Will incorporate this.